tests/sentry/replays/unit/test_issue_creation.py

```diff
@@ -235,3 +238,79 @@ def test_report_rage_click_no_component_name(mock_new_issue_occurrence, default_
+def test_report_hydration_error_issue_with_replay_event(mock_new_issue_occurrence, default_project):
+    seq1_timestamp = datetime.now() - timedelta(minutes=10, seconds=52)
+
+    replay_id = "b58a67446c914f44a4e329763420047b"
+    report_hydration_error_issue_with_replay_event(
+        project_id=default_project.id,
+        replay_id=replay_id,
+        timestamp=seq1_timestamp.timestamp(),
+        url="https://www.sentry.io",
+        replay_event=mock_replay_event(),
+    )
+    issue_occurence_call = mock_new_issue_occurrence.call_args[1]
+    assert issue_occurence_call["culprit"] == "https://www.sentry.io"
+    assert issue_occurence_call["environment"] == "production"
+    # assert issue_occurence_call["fingerprint"] == ["div.xyz > SmartSearchBar"]  # TODO:
+    assert issue_occurence_call["issue_type"].type_id == 5003
+    assert issue_occurence_call["level"] == "error"
+    assert issue_occurence_call["platform"] == "javascript"
+    assert issue_occurence_call["project_id"] == default_project.id
+    assert issue_occurence_call["title"] == "Hydration Error"
+
+    assert issue_occurence_call["extra_event_data"] == {
+        "contexts": {
+            "browser": {"name": "Chrome", "version": "103.0.38"},
+            "device": {
+                "brand": "Apple",
+                "family": "iPhone",
+                "model": "13 Pro",
+                "name": "iPhone 13 Pro",
+            },
+            "os": {"name": "iOS", "version": "16.2"},
+            "replay": {"replay_id": "b58a67446c914f44a4e329763420047b"},
+            "trace": {
+                "trace_id": "4491657243ba4dbebd2f6bd62b733080",
+            },
+        },
+        "dist": "abc123",
+        "level": "error",
+        "release": "version@1.3",
+        "sdk": {"name": "sentry.javascript.react", "version": "6.18.1"},
+        "tags": {
+            "replayId": "b58a67446c914f44a4e329763420047b",
+            "transaction": "Title",
+            "url": "https://www.sentry.io",
+        },
+        "user": {
+            "email": "test@test.com",
+            "id": "1",
+            "ip_address": "127.0.0.1",
+            "username": "username",
+        },
+    }
+
+
+@pytest.mark.snuba
+@django_db_all
+def test_report_hydration_error_long_url(default_project):
+    replay_id = "b58a67446c914f44a4e329763420047b"
+    seq1_timestamp = datetime.now() - timedelta(minutes=10, seconds=52)
+    report_hydration_error_issue_with_replay_event(
+        project_id=default_project.id,
+        replay_id=replay_id,
+        timestamp=seq1_timestamp.timestamp(),
+        url=f"https://www.sentry.io{'a' * 300}",
+        replay_event=mock_replay_event(),
+    )
+
+    # test that the Issue gets created with the truncated url
+    # WIP: these asserts fail (no groups created)
+
+    # assert Group.objects.all().count()  # .get(culprit__contains="www.sentry.io")
+    # assert Group.objects.get(message__contains="Hydration Error")
```

Comment:
Does this need to be removed?

Reply:
Its a test I was trying to get to pass yesterday, so I commented and marked as draft